scripts/auth0connections/auth0connections.py

```diff
@@ -65,13 +84,19 @@ def cli(ctx, config_file):
 
 
 @cli.command()
-def remote():
+@click.option('--names', '-n', help='Only print each connection\'s name', is_flag=True)
+def remote(names):
     """
     Show a list of existing connections on auth0
     """
     click.echo("Remote connections:")
     client = get_client()
-    click.echo(yaml.safe_dump(client.connections.all()))
+    if names:
+        click.echo(yaml.safe_dump(
+            [c['name'] for c in client.connections.all()]
+        ))
+    else:
+        click.echo(yaml.safe_dump(client.connections.all()))
```

Comment:
Not a big deal but there is some duplication between the `if` and the `else` branches, maybe you could write something like this (just an idea):

```python
[...]
connections = client.connections.all()
if names:
    connections = [ c["name"] for c in connections ]

click.echo(yaml.safe_dump(connections))
```